src/DynamoPackages/PackageManagerClient.cs

```diff
@@ -65,6 +65,18 @@ internal bool Upvote(string packageId)
             }, false);
         }
 
+        internal List<string> UserVotes()
+        {
+            var votes = FailFunc.TryExecute(() =>
+            {
+                var nv = new GetUserVotes();
+                var pkgResponse = this.client.ExecuteAndDeserializeWithContent<UserVotes>(nv);
+                return pkgResponse.content;
+            }, null);
+
+            return votes.has_upvoted;
```

Comment:
the local variable votes will be null when the function fails. Check for null before returning its property.